test/gtest/mp/CMakeLists.txt

```diff
@@ -16,10 +16,10 @@ add_executable(
   ../common/drop.cpp
   ../common/enumerate.cpp
   ../common/equal.cpp
-  ../common/exclusive_scan.cpp
+#  ../common/exclusive_scan.cpp
```

Comment:
please report issue in JIRA and put its it here and near inclusive scan with some info what is the problem (log snippet with failure, link to failed execution)

Reply:
Done